protocol/common/constants.go

```diff
@@ -21,48 +32,76 @@ const (
 
 var (
 	// Topic names and UUIDs - initialized in init()
-	TOPIC1_NAME       string
-	TOPIC2_NAME       string
-	TOPIC3_NAME       string
-	TOPIC1_UUID       string
-	TOPIC2_UUID       string
-	TOPIC3_UUID       string
-	TOPICX_UUID       string
-	TOPIC_UNKOWN_NAME string
-	TOPIC_UNKOWN_UUID string
+	TOPIC1_NAME, TOPIC2_NAME, TOPIC3_NAME, TOPIC4_NAME string
+	TOPIC1_UUID, TOPIC2_UUID, TOPIC3_UUID, TOPIC4_UUID string
+	TOPICX_UUID                                        string
+	TOPIC_UNKOWN_NAME, TOPIC_UNKOWN_UUID               string
 
-	MESSAGE1 string
-	MESSAGE2 string
-	MESSAGE3 string
+	MESSAGE1, MESSAGE2, MESSAGE3 string
+	TOPICS                       map[string]TopicConfig
 )
 
 func init() {
 	random.Init()
 
 	all_topic_names := []string{"foo", "bar", "baz", "qux", "quz", "pax", "paz", "saz"}
-	topic_names := GetSortedValues(random.RandomElementsFromArray(all_topic_names, 4))
+	topic_names := getSortedValues(random.RandomElementsFromArray(all_topic_names, 5))
 	TOPIC1_NAME = topic_names[0]
 	TOPIC2_NAME = topic_names[1]
 	TOPIC3_NAME = topic_names[2]
-	random_topic_uuids := random.RandomInts(10, 100, 3)
+	TOPIC4_NAME = topic_names[3]
+	random_topic_uuids := random.RandomInts(10, 100, 4)
```

Comment:
Let's avoid "pre-creating" these randomly here and stick to our usual pattern of creating random values on the fly in each stage. You can expose something like `getRandomTopicConfig()` / `getRandomTopicConfigs()` if you'd like and use that. 

From a user's perspective, let's have the stage functions read more cleanly and make it clear that values are generated randomly on the fly.